run/commentbot.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
meta-suggestion: these should technically be `#!/usr/bin/env sh`